.github/workflows/build.yaml

```diff
@@ -0,0 +1,71 @@
+name: Build
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
ubuntu-latest is soon going to move to ubuntu-20.04. When that happens `shivammathur/setup-php@v2` will not
be able to install php 7.2 anymore.

Reply:
@christiaan It will install PHP 7.2, will just take some time to install. 🙂

Reply:
Thanks that is good to know 😄.